tests/test_cxl_host.py

```diff
@@ -545,109 +545,110 @@ async def test_cxl_host_type3_ete():
 #     await asyncio.gather(*start_tasks)
 
 
-@pytest.mark.asyncio
-async def test_cxl_host_type3_complex_host_ete():
-    # pylint: disable=protected-access
-    host_port = BASE_TEST_PORT + pytest.PORT.TEST_5 + 155
-    util_port = BASE_TEST_PORT + pytest.PORT.TEST_5 + 156
-    switch_port = BASE_TEST_PORT + pytest.PORT.TEST_5 + 157
+# NOTE: Temporarily commented out due to changes in PCI Bus Driver
+# @pytest.mark.asyncio
+# async def test_cxl_host_type3_complex_host_ete():
```

Comment:
Could you file a ticket to track this?

Reply:
https://github.com/opencxl/opencxl/issues/79